spec/util/test.js

```diff
@@ -76,5 +76,9 @@ module.exports.beforeEach = function(testCase, env) {
         });
         env.mockAsapiController = env.asapiMock.create();
     }
+
+    process.on("unhandledRejection", function(reason, promise) {
+        throw new Error("Unhandled rejection: " + reason);
```

Comment:
The stack trace is useless, but the error bubbles up to the top level, which is the important thing.

I'm not sure how to get a better stack trace, given this is all happening in an event loop...